index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <title>Awesome books</title>
+</head>
+<body>
+    <nav class="position-absolute top-0 start-50 translate-middle-x">
+        <h1>Awesome books</h1>
+    </nav>
+    <main>
+        <div class="container position-absolute top-50 start-0 translate-middle-y">
+            <div id="alert">
+
+            </div>
+            <div class="row">
+                <div class="col">
+                    <ul id="ul"class="list-group">
+
+                    </ul>
+                </div>
+                <div class="col">
+                    <form id='form' class="shadow-lg p-3 mb-5 bg-body rounded">
+                        <input type="text" class="form-control" id="title" placeholder="Title"><br>
+                        <input type="text" class="form-control" id="author"  placeholder="Author"><br>
+                        <button type="submit" id="btn" class="btn btn-primary w-100">Add</button>
+                
+                    </form>
+                </div>
+            </div>
+        </div>
+    </main> 
+    <script src="./index.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js" integrity="sha384-IQsoLXl5PILFhosVNubq5LC7Qb9DXgDA9i+tQ8Zj3iwWAwPtgFTxbJ8NT4GN1R8p" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.min.js" integrity="sha384-cVKIPhGWiC2Al4u+LWgxfKTRIcfu0JTxR+EQDz/bgldoEyl4H0zUF0QKbrJ0EcQF" crossorigin="anonymous"></script>
```

Comment:
Good job showing your bootstrap skills but for now, the project instructions require you not to use framework or library. 

I think you missed this 👉  `Don't use any library or framework, use only plain JavaScript.` when you were going through the [Project Requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md)